ambry-api/src/main/java/com.github.ambry/router/GetBlobOptions.java

```diff
@@ -109,6 +109,8 @@ public int hashCode() {
     Data, /**
      * Return just the blob info in the response.
      */
-    BlobInfo;
+    BlobInfo,
+
+    BlobChunkIds
```

Comment:
add a javadoc here. Also, unrelated to your change, but while your at it could you add newlines in between each enum entry above ^? Otherwise the autoformatter does weird stuff like that.